index.bs

```diff
@@ -379,20 +379,68 @@ If a document's [=document/embedder policy=] is "`require-corp`", then any docum
 [[#cascade-vs-require]]).
 
 To <dfn abstract-op lt="process navigation response">check a navigation response's adherence to its
-embedder's policy</dfn> given a [=response=] (|response|), and a target [=browsing context=]
-(|target|), execute the following steps, which will return "`Allowed`" or "`Blocked`" as
-appropriate:
+embedder's policy</dfn> given a [=request=] (|request|), a [=response=] (|response|), and a target
+[=browsing context=] (|target|), execute the following steps, which will return "`Allowed`" or
+"`Blocked`" as appropriate:
 
-1.  Let |response policy| be the result of [$parse header|obtaining an embedder policy$] from
+1.  Return "`Allowed`" if |target| is not a [=child browsing context=].
+
+2.  Let |response policy| be the result of [$parse header|obtaining an embedder policy$] from
     |response|.
 
-2.  Return "`Allowed`" if any of the following statements are true:
+3.  Let |parent policy| be |target|'s [=container document=]'s [=document/embedder policy=].
+
+4.  If |parent policy|'s [=embedder policy/report only value=] is "`require-corp`" and
+       |parent policy|'s [=embedder policy/report only reporting endpoint=] is not null and
+       |response policy|'s [=embedder policy/value=] is "`unsafe-none`":
+
+    1.  Let |blocked url| be |request|'s [=request/URL=].
```

Comment:
Perhaps add a note here that this is intentionally not request's current URL.

Reply:
Done.